saleor/product/filters.py

```diff
@@ -51,12 +51,14 @@ def _get_attributes(self):
         product_attributes = (
             Attribute.objects.all()
             .prefetch_related("translations", "values__translations")
+            .exclude(visible_in_storefront=False)
```

Comment:
I think here and below we should also take into account attributes with `filterable_in_storefront=True` property.